.github/workflows/meson.yml

```diff
@@ -86,6 +95,13 @@ jobs:
               $pre-fribidi $pre-freetype $pre-harfbuzz \
               $pre-fontconfig $pre-libpng
             echo "PKG_CONFIG_PATH=$(cygpath -w "$PWD/pacman-root/mingw64/lib/pkgconfig")" >> "$GITHUB_ENV"
+            cygpath -w "$PWD/pacman-root/mingw64/bin" >> "$GITHUB_PATH"
+            cp "$PWD/pacman-root/mingw64/lib/libharfbuzz.dll.a" "$PWD/pacman-root/mingw64/lib/libharfbuzz.lib"
+            cp "$PWD/pacman-root/mingw64/lib/libfreetype.dll.a" "$PWD/pacman-root/mingw64/lib/libfreetype.lib"
+            cp "$PWD/pacman-root/mingw64/lib/libfribidi.dll.a" "$PWD/pacman-root/mingw64/lib/libfribidi.lib"
+            cp "$PWD/pacman-root/mingw64/lib/libfontconfig.dll.a" "$PWD/pacman-root/mingw64/lib/libfontconfig.lib"
+            cp "$PWD/pacman-root/mingw64/lib/libpng.dll.a" "$PWD/pacman-root/mingw64/lib/libpng.lib"
+            cp "$PWD/pacman-root/mingw64/lib/libpng16.dll.a" "$PWD/pacman-root/mingw64/lib/libpng16.lib"
```

Comment:
Mixing mingw build with msvc one defeats the purpose, no? (and is generally unsafe, if public headers are not carefully vetted to work) All those libraries are available in meson's wrap db which make it very easy to build when needed.

Reply:
Putting aside whether this mixing is in general safe or not, the build setup is more complex yet slower than wrapdb+sccache from #857. The latter brought down MSVC build times into the range of other Windows jobs, while the mixed setup here sticks out taking ~60% more time than the next slowest CI job.

Perhaps the argument can be made for there being some merit in having this setup in git history as a point of reference for what it takes to get a mixed setup working; but for further use sccache+wrapdb just seems more beneficial to me